MultithreadedProgram/ThreadBuilder.cs

```diff
@@ -7,13 +7,18 @@ public class ThreadBuilder
     {
         public static void BuildThread()
         {
-            char randomSymbol = RandomHelper.GetRandomSymbol();
-            ConsoleColor randomColor = RandomHelper.GetRandomColor();
-            int randomDelay = RandomHelper.GetRandomDelay();
+            while (true)
+            {
+                char randomSymbol = RandomHelper.GetRandomSymbol();
+                ConsoleColor randomColor = RandomHelper.GetRandomColor();
+                int randomDelay = RandomHelper.GetRandomDelay();
+
+                Writer writer = new Writer(randomSymbol, randomColor);
+                Thread thread = new Thread(writer.Write);
+                thread.Start();
+                Thread.Sleep(randomDelay);
```

Comment:
randomDelay shouldn't be here. It's a property bound to Writer class. Also make ThreadBuilder class static